castle/cms/cron/_social_counts.py

```diff
@@ -213,7 +213,6 @@ def get_social_counts(site, obj, site_url, count_types, count=0):
 
     transaction.commit()
 
-    index_in_es(obj)
```

Comment:
This method was moved to: https://github.com/castlecms/castle.cms/blob/5c859a61bcc7550e51e3a0f6f0e9511f2581dfaa/castle/cms/indexing/hps.py#L159

However, this cron script probably needs additional changes to ensure that it uses WildcardHPS, or maybe a plone interface of some sort, to reindex an object in the catalog, because I don't think this is going to index the object in a custom index for this script, right? it'll just reindex the object in the normal catalog?

either way, update the location of the method here, but don't remove it outright for now. We'll create a different task to fix this.